.github/workflows/ci.yml

```diff
@@ -82,12 +82,20 @@ jobs:
         if: matrix.java == 'corretto@21' && steps.setup-java-corretto-21.outputs.cache-hit == 'false'
         run: sbt +update
 
+      - name: gcloud auth
+        if: (github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository)
```

Comment:
we could also just check if `!pull_request.head.repo.fork` - wdyt?

Reply:
Thanks!